_config.yml

```diff
@@ -34,14 +34,15 @@ plugins:
   - jekyll-feed
   - jekyll-seo-tag
 
-# mapbox access token
-# TODO: use config file for map center
+
 mapbox_access_token: pk.eyJ1IjoibWlrbGItYzR0YiIsImEiOiJjam5kOGg3c2kwMDB3M3dwYXQyMDJpdzFtIn0.yKOyo193qF1quWsNwiFzug
 
-shelter_map_center: -88.9159, 35.8198
+shelter_map_center: -95.7129, 37.0902
+shelter_zoom: 3.6
 
-pod_map_center: -88.9159, 35.8198
+pod_map_center: -95.7129, 7.0902
```

Comment:
This center point doesn't match shelters  -- is that right?

Reply:
Likely should be `37.0902` instead of `7.0902`

Reply:
The default center I used was middle of US and a lower zoom so when there isn't an event we can pick up any open shelters for development. For an event, we can adjust the centers and zoom in the config.

Reply:
yep, sorry.